web/controllers/wanikani.js

```diff
@@ -0,0 +1,60 @@
+
+const express = require('express')
+const router = express.Router();
+const wk = require('../../services/wanikani.js')
+
+// TODO: refactor this out! /////////////////////
+const db = require("../../database.js")
+function dateString(d) {
+	let mm = d.getMonth() + 1
+	if(mm < 10) { mm = "0" + mm }
+	let dd = d.getDate()
+	if(dd < 10) { dd = "0" + dd }
+	return d.getFullYear() + "-" + mm + "-" + dd
+}
+async function waniGetTokens() {
+	return db.query(`SELECT * FROM wanikani`).then((response) => {
+		return response
+  })
+}
+function waniGetData(users) {
```

Comment:
should this be part of the wanikani service module? or do you want to leave that one as being strictly an api wrapper?